Makefile

```diff
@@ -1,8 +1,8 @@
 CHS_ENV_HOME ?= $(HOME)/.chs_env
 TESTS        ?= ./...
 
-bin          := lfp-pay-api
-chs_envs     := $(CHS_ENV_HOME)/global_env $(CHS_ENV_HOME)/lfp-pay-api/env
+bin          := penalty-payment-api
+chs_envs     := $(CHS_ENV_HOME)/global_env $(CHS_ENV_HOME)/penalty-payment-api/env
```

Comment:
This is another Env Variable thing that I don't think we can change yet - it looks to relate to chs-configs per team / shared chs environments.  See e.g. https://github.com/companieshouse/chs-configs/blob/develop/cidev/lfp-pay-api/env

Reply:
Will update